### DISCUSSION TITLE
Some Ideas

### DISCUSSION THREAD

**Discussion by meteorquake** [LibreHardwareMonitor/LibreHardwareMonitor#956] [Ideas]
Good morning! I've just installed this program after I googled alternatives to OpenHardwareMonitor, as this project seems updated. So I have some suggestions from things that came to mind, to make it even better...
* Free RAM tray icon should show 1 decimal because 0 or 1 isn't greatly helpful as a value. Presumably other RAM items could be set too. It may be this could be done with a right-click of tray icon where you can opt to show one decimal for the display value or not, and/or it can be done by a right-click in the GUI list.
* Whatever your tray icon setting and display, hovering over the icon should show the full-detail value, currently it just shows the overly-coarse rounded value.
* It would be good to have thresholds above/below which the tray icon changes colour e.g. beyond the value one could set it to red. Perhaps there could be a global warning colour and you set the thresholds for it to change to that colour. Perhaps settings for those that want good values to change gradually to that colour.
* In showing hiding entries in the main GUI you have to right-click and Hide. It would be good to be make this easier for example Ctrl-Click or similar could toggle the hidden status on/off. The Right-Click - Hide menu could indicate this method.
* Memory use of Libre Hardware Monitor is double that of Open Hardware Monitor. It would be good to see if there are simple ways for reducing this.
* Help menu could have more than about, for example an item for Feedback/Issues that takes to the discussion forum, one for the latest release, and so on. This is better and more accessible I think than having a link in the About dialog.
Cheers! David

**Comment by YarekTyshchenko** [comment]
> Memory use of Libre Hardware Monitor is double that of Open Hardware Monitor. It would be good to see if there are simple ways for reducing this.

I haven't benchmarked it but mine uses ~44MB of ram, seems low enough that no-one should care these days
